pkgs/by-name/si/sitespeed-io/package.nix

```diff
@@ -46,9 +44,13 @@ buildNpmPackage rec {
   GECKODRIVER_SKIP_DOWNLOAD = true;
   EDGEDRIVER_SKIP_DOWNLOAD = true;
 
+  buildInputs = [
+    systemdLibs
```

Comment:
I added this, since without it the build failed, complaining about `libudev.h` missing. I'm unsure whether this is the correct way to do this.